packages/now-cli/test/unit.js

```diff
@@ -2,389 +2,21 @@ import { basename, join, sep } from 'path';
-  t.is(base(files[1]), 'files-overrides-gitignore/test.js');
-  t.is(base(files[2]), 'files-overrides-gitignore/test.json');
-});
-
-test('`now.files` overrides `.gitignore` in Node', async t => {
-  const path = 'now-json-npm-gitignore-override';
-  let files = await getNpmFiles(
-    fixture(path),
-    await loadJSON(getLocalConfigPath(fixture(path)))
-  );
-  files = files.sort(alpha);
-
-  t.is(files.length, 5);
-  t.is(base(files[0]), `${path}/a.js`);
-  t.is(base(files[1]), `${path}/b.js`);
-  t.is(base(files[2]), `${path}/build/a/c.js`);
-  t.is(base(files[3]), `${path}/now.json`);
-  t.is(base(files[4]), `${path}/package.json`);
-});
-
-test('`now.files` overrides `.npmignore` in Node', async t => {
-  const path = 'now-json-npm-npmignore-override';
-  let files = await getNpmFiles(
-    fixture(path),
-    await loadJSON(getLocalConfigPath(fixture(path)))
-  );
-  files = files.sort(alpha);
-
-  t.is(files.length, 6);
-  t.is(base(files[0]), `${path}/a.js`);
-  t.is(base(files[1]), `${path}/b.js`);
-  t.is(base(files[2]), `${path}/build/a/c.js`);
-  t.is(base(files[3]), `${path}/c.js`);
-  t.is(base(files[4]), `${path}/now.json`);
-  t.is(base(files[5]), `${path}/package.json`);
-});
-
-test('`now.files` overrides `.gitignore` in Static with custom config path', async t => {
-  const path = 'now-json-static-gitignore-override';
-
-  // Simulate custom args passed by the user
-  process.argv = [...process.argv, '--local-config', './now.json'];
-
-  let files = await getStaticFiles(fixture(path));
-
-  files = files.sort(alpha);
-
-  t.is(files.length, 3);
-  t.is(base(files[0]), `${path}/a.js`);
-  t.is(base(files[1]), `${path}/b.js`);
-  t.is(base(files[2]), `${path}/build/a/c.js`);
-});
-
-test('`now.files` overrides `.gitignore` in Static', async t => {
-  const path = 'now-json-static-gitignore-override';
-  let files = await getStaticFiles(fixture(path));
-  files = files.sort(alpha);
-
-  t.is(files.length, 3);
-  t.is(base(files[0]), `${path}/a.js`);
-  t.is(base(files[1]), `${path}/b.js`);
-  t.is(base(files[2]), `${path}/build/a/c.js`);
-});
-
-test('discover static files without `now.files`', async t => {
-  const path = 'now-json-static-no-files';
-  let files = await getStaticFiles(fixture(path));
-  files = files.sort(alpha);
-
-  t.is(files.length, 4);
-
-  t.is(base(files[0]), `${path}/a.js`);
-  t.is(base(files[1]), `${path}/b.js`);
-  t.is(base(files[2]), `${path}/build/a/c.js`);
-  t.is(base(files[3]), `${path}/package.json`);
-});
-
-test('discover files for builds deployment', async t => {
-  const path = 'now-json-static-no-files';
-  let files = await getStaticFiles(fixture(path));
-  files = files.sort(alpha);
-
-  t.is(files.length, 4);
-
-  t.is(base(files[0]), `${path}/a.js`);
-  t.is(base(files[1]), `${path}/b.js`);
-  t.is(base(files[2]), `${path}/build/a/c.js`);
-  t.is(base(files[3]), `${path}/package.json`);
-});
-
-test('should observe .vercelignore file', async t => {
```

Comment:
We need to keep tests for `getStaticFiles()` which is still used for v2

Reply:
I re-added the vercelignore test